hadoop-project/pom.xml

```diff
@@ -167,8 +167,14 @@
     <enforced.java.version>[${javac.version},)</enforced.java.version>
     <enforced.maven.version>[3.3.0,)</enforced.maven.version>
 
+    <extraJavaTestArgs>
```

Comment:
I haven't decided on the name for this tag yet, but I feel that calling it `extraJavaTestArgs` may not be the best option.

Reply:
the name is borrowed from the Spark project.

Reply:
This is not a blocking concern, as it relates to JDK-specific parameters. I'm unsure whether it should be named something related to the JDK. I have no objections if we keep it as is.